ccc-js/src/main/javascript/package-res/ccc/core/cartesian/axis/cart-axis.js

```diff
@@ -963,6 +964,12 @@ pvc_CartesianAxis.options({
         resolve: '_resolveFull'
     },
 
+    TicksRegularity: {
```

Comment:
I'd go for something like `PreserveTickAlignment`. Also, this option should be set/defaulted to true when the CCC sliding window is used.